docs/topics/db/aggregation.txt

```diff
@@ -632,3 +634,36 @@ aggregate that author count, referencing the annotation field:
     >>> from django.db.models import Avg, Count
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
+
+Handling empty querysets and groups using aggregate ``default``
+---------------------------------------------------------------
+
+When an aggregate function is used with an empty ``QuerySet`` or grouping,
+``None`` is returned as a default. This is because aggregate functions result
+in ``NULL`` when used with an empty result set. The optional ``default``
+argument for aggregate functions allows specifying a value to use as a
+default when the executed query returns no rows. Some aggregate functions
+do not support the ``default`` argument because they already handle this, e.g.
+``Count`` returns ``0``.
+
+For example, to get the total price of all books containing
+"web" in name. Without the ``default`` argument, if there are no books
+returned by the query, and there are no rows to compute the aggregate
+``Sum``, None is returned by default.
+
+.. code-block:: pycon
+
+    >>> from django.db.models import Sum
+    >>> Book.objects.filter(name__contains="web").aggregate(Sum("price"))
+    {"price__sum": None}
+
+However, you can pass ``0`` as a ``default`` to be returned if no books
+can be found.
+
+.. code-block:: pycon
+
+    >>> Book.objects.filter(name__contains="web").aggregate(Sum("price", default=0))
+    {"price__sum": Decimal("0")}
+
+The ``default`` argument is implemented under the hood by wrapping the
```

Comment:
I'm not sure this paragraph makes sense in a "topic" page, where concepts are a bit higher level than references.

Do you think this is relevant for the reader? If yes, why so? Thanks!

Reply:
Hey @nessita , IMO the reference would be necessary for comparison in approach because initially the idea was to document "Coalesce", the "default argument" was suggested as a cleaner option in the ticket.